src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -1651,10 +1651,17 @@ static void uct_ib_mlx5_devx_check_odp(uct_ib_mlx5_md_t *md,
         goto no_odp;
     }
 
-    ucs_debug("%s: ODP is supported, version %d",
-              uct_ib_device_name(&md->super.dev), version);
-    md->super.reg_nonblock_mem_types = UCS_BIT(UCS_MEMORY_TYPE_HOST);
-    md->super.gva_mem_types          = UCS_BIT(UCS_MEMORY_TYPE_HOST);
+    md->super.gva_mem_types           = UCS_BIT(UCS_MEMORY_TYPE_HOST);
```

Comment:
testing remains for GVA

Reply:
tested with setting `.gva_mem_types` with cuda-managed and host on a Grace:
- `UCX_GVA_ENABLE=n` no call to `uct_ib_mlx5_devx_mem_reg_gva()`
- `UCX_GVA_ENABLE=y` successful call to `uct_ib_mlx5_devx_mem_reg_gva()`, if and only if cuda-managed is found in `.gva_mem_types`